src/function_spaces.jl

```diff
@@ -87,8 +103,8 @@ function derivative!{T}(fs::Lagrange{1, RefCube, 1}, dN::Vector{Vec{1, T}}, ξ::
 end
 
 function reference_coordinates(fs::Lagrange{1, RefCube, 1})
-    return (Vec{1, Float64}((-1.0,)),
-            Vec{1, Float64}(( 1.0,)))
+    return [Vec{1, Float64}((-1.0,)),
```

Comment:
Sure, there are a number of other functions which are only used for testing. We can extract them all to a `test_utils.jl` file or something.